cli/compose/schema/data/config_schema_v3.3.json

```diff
@@ -0,0 +1,475 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "config_schema_v3.2.json",
```

Comment:
Please note that this PR is the strict equivalent of what was original proposed. This line is wrong and would have been fixed with moby/moby#32743